app/models/aspace_svn/get_eads_job.rb

```diff
@@ -0,0 +1,99 @@
+require 'archivesspace/client'
+require 'nokogiri'
+require 'fileutils'
+
+module AspaceSvn
+  class GetEadsJob < LibJob
+    def initialize()
+      super(category:"EAD_export")
+      @errors = []
+    end
+    def aspace_login()
+      #configure access
+      @config = ArchivesSpace::Configuration.new({
+        base_uri: ENV['ASPACE_URL'],
+        base_repo: "",
+        username: ENV['ASPACE_USER'],
+        password: ENV['ASPACE_PASSWORD'],
+        #page_size: 50,
+        throttle: 0,
+        verify_ssl: false
+      })
+      #log in
+      @client = ArchivesSpace::Client.new(@config).login
+    end
+    def handle(data_set:)
+      aspace_login
+      repos = (3..12).to_a
+      repos.each do |repo|
+
+        # make directories if they don't already exist
+        dir =
+          case repo
+            when 3
+              FileUtils.mkdir_p("mudd/publicpolicy") unless Dir.exists?("mudd/publicpolicy")
```

Comment:
One possible simplification: would it behave the same if you remove all `FileUtils` and `Dir` lines from the `case` statement, and then just add the following after the `end` of the `case` statement?

```
FileUtils.mkdir_p(dir) unless Dir.exists?(dir)
```

Reply:
It does! Much cleaner. Thanks @sandbergja !